pwa/src/components/PatientInfoForm.tsx

```diff
@@ -0,0 +1,265 @@
+  const [planning, setPlanning] = useState(0); // Default: No
+
+  // Data for Radio Buttons
+  const sexValues = [
+    { value: 0, label: "Male" },
+    { value: 1, label: "Female" },
+  ];
+
+  const radioYesNo = [
+    { value: 0, label: "No" },
+    { value: 1, label: "Yes" },
+  ];
+
+  /* istanbul ignore next */
+  const onSubmit = () => {
+    // This shows how to access the values of the form
+    const firstNameField: HTMLInputElement = document.getElementById(
+      "firstname"
+    ) as HTMLInputElement;
+    const lastNameField: HTMLInputElement = document.getElementById(
+      "lastname"
+    ) as HTMLInputElement;
+    const nicknameField: HTMLInputElement = document.getElementById(
+      "nickname"
+    ) as HTMLInputElement;
+    const birthdateField: HTMLInputElement = document.getElementById(
+      "birthdate"
+    ) as HTMLInputElement;
+    const complaintField1: HTMLInputElement = document.getElementById(
+      "complaint1"
+    ) as HTMLInputElement;
+    const complaintField2: HTMLInputElement = document.getElementById(
+      "complaint2"
+    ) as HTMLInputElement;
+    const complaintField3: HTMLInputElement = document.getElementById(
+      "complaint3"
+    ) as HTMLInputElement;
+
+    console.log("First name:", firstNameField.value);
+    console.log("Last name:", lastNameField.value);
+    console.log("Nickname:", nicknameField.value);
+    console.log("Birth Date:", birthdateField.value);
+    console.log(
+      "Complaints: ",
+      complaintField1.value +
+        " " +
+        complaintField2.value +
+        " " +
+        complaintField3.value
+    );
+    console.log("Wants Planning?: " + planning);
+  };
+
+  return (
+    <div id="patient-info-wrapper">
+      <TextareaItem
+        title="First Name"
+        placeholder="First Name"
+        autoHeight
+        id="firstname"
+      />
+
+      <TextareaItem
+        title="Last Name"
+        placeholder="Last Name"
+        id="lastname"
+        autoHeight
+      />
+
+      <TextareaItem
+        title="Nickname"
+        placeholder="Nickname"
+        id="nickname"
+        autoHeight
+      />
+
+      <List renderHeader={() => "Sex"}>
+        {sexValues.map((i) => (
+          <RadioItem
+            data-testid={`${i.label.toLowerCase()}-button`}
+            id={`${i.label.toLowerCase()}-button`}
+            key={i.value}
+            name="sex"
+            checked={sex === i.value}
+            onClick={() => setSex(i.value)}
+          >
+            {i.label}
+          </RadioItem>
+        ))}
+      </List>
+
+      <WhiteSpace />
+
+      <InputItem placeholder="YYYY-MM-DD" id="birthdate">
+        Birth Date
+      </InputItem>
+
+      <InputItem type="number" placeholder="Age" id="age">
+        Age{" "}
+      </InputItem>
+
+      <InputItem type="number" placeholder="Phone" id="phone">
+        Phone
+      </InputItem>
+
+      <TextareaItem title="Town" placeholder="Town" id="town" autoHeight />
```

Comment:
This will eventually be replaced with a searchable multi-select from the town database correct?

Reply:
Added issue #72 to get access to this data in case we have time to integrate it in the next 3 weeks.

Reply:
@meghanmc feel free to resolve this anytime as it is not necessary for this PR